packages/notebook/src/celllist.ts

```diff
@@ -182,8 +144,8 @@ class CellList implements IObservableUndoableVector<ICellModel> {
    * #### Undefined Behavior
    * An `index` which is non-integral or out of range.
    */
-  at(index: number): ICellModel {
-    return this._cellMap.get(this._cellOrder.at(index)) as ICellModel;
+  get(index: number): ICellModel {
+    return this._cellMap.get(this._cellOrder.get(index)) as ICellModel;
```

Comment:
At least with the Google Drive implementation, this appears to be enough (the documentation is fairly light on that front). We may need to revisit it for other backends.